[2024-05-29T17:02:14.412+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T17:02:14.831+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-05-29T17:02:15.066+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-05-29T17:02:15.089+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T17:02:15.557+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_2> on 2024-02-28 00:00:00+00:00
[2024-05-29T17:02:15.640+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=218) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T17:02:15.672+0000] {standard_task_runner.py:63} INFO - Started process 257 to run task
[2024-05-29T17:02:15.653+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_1', 'task_2', 'scheduled__2024-02-28T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpuram55yf']
[2024-05-29T17:02:15.683+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask task_2
[2024-05-29T17:02:16.976+0000] {task_command.py:426} INFO - Running <TaskInstance: test_1.task_2 scheduled__2024-02-28T00:00:00+00:00 [running]> on host 2832635d2b45
[2024-05-29T17:02:18.518+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_1' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-28T00:00:00+00:00'
[2024-05-29T17:02:18.543+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T17:02:18.558+0000] {logging_mixin.py:188} INFO - Run function_2!
[2024-05-29T17:02:18.561+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T17:02:18.579+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T17:02:18.663+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_1, task_id=task_2, run_id=scheduled__2024-02-28T00:00:00+00:00, execution_date=20240228T000000, start_date=20240529T170214, end_date=20240529T170218
[2024-05-29T17:02:19.140+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T17:02:19.482+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-29T17:02:19.501+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
